cpu/sam0_common/periph/gpio.c

```diff
@@ -109,12 +109,17 @@ static inline PortGroup *_port_iobus(gpio_t pin)
 
 static inline PortGroup *_port(gpio_t pin)
 {
-#ifdef PORT_IOBUS
+#ifdef MODULE_PERIPH_GPIO_FAST_READ
     /* Shift the PortGroup address back from the IOBUS region to the peripheral
      * region
      */
+#if !defined(CPU_FAM_SAML11)
```

Comment:
```suggestion
#if !defined(PORT_IOBUS_SEC)
```

but I'd also try to avoid the inverse logic, so

```C
#if defined(PORT_IOBUS_SEC)
    return (PortGroup *)((uintptr_t)_port_iobus(pin) -
                         (uintptr_t)PORT_IOBUS_SEC + (uintptr_t)PORT_SEC);
#else
    return (PortGroup *)((uintptr_t)_port_iobus(pin) -
                         (uintptr_t)PORT_IOBUS + (uintptr_t)PORT);
#endif
```